### ISSUE TITLE
[Visual Requirements – Make Code – Zoom In] Controls are not visible properly after resizing the page up to 200%.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3625]
**User Experience:**
If, the content of the page does not appear properly after resizing it up to 200%, then the user who depends upon keyboard for navigation might get affected and failed to get the proper information of the page content.

**Note:** User credentials should NOT be included in the bug.
**Repro Steps:**
1. Launch application URL :
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Undo/Redo/Zoom In/Zoom Out" button and select it.
6. Verify user able to perform all action using the controls.
7. Verify the issue.

**Actual Result:**
 Controls like zoom in, zoom out are not visible after resizing the page up to 200%.

**Expected Result:**
Controls like zoom in, zoom out should be visible properly after resizing the page up to 200%.

**Note:**
'Explorer' button is not visible after resizing the page up to 200%.

**WCAG Reference:**
[WCAG 1.4.4 - Resize Text](https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html)

**Test Environments:**
Version: 2004 (OS Build: 19041.572)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev1, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.4.4, #Win10-Edge(Chromium)

<img width="1903" alt="WCAG_1 4 4_when page is resize 200% Zoom In,Zoom out control are not visible beside undo,redo control" src="https://user-images.githubusercontent.com/72871089/98523906-81089d00-229c-11eb-97da-3694bf8a308f.png">

**Comment by jwunderl** [comment]
Some secondary buttons that are unnecessary / redundant get removed as viewport shrinks for support on smaller screens. Download button is visible in the image above (icon only). zoom in / out can be accomplished with standard keyboard shortcuts.
